src/legacy/core_plugins/kbn_vislib_vis_types/public/editors/point_series.tsx

```diff
@@ -0,0 +1,91 @@
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import { EuiPanel, EuiTitle, EuiSpacer } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+
+import { VisOptionsProps } from 'ui/vis/editors/default';
+import { SwitchOption } from '../controls/switch';
+import { GridOptions } from '../controls/point_series/grid_options';
+import { BasicOptions } from '../controls/basic_options';
+import { BasicVislibParams } from '../types';
+
+function PointSeriesOptions(props: VisOptionsProps<BasicVislibParams>) {
+  const { stateParams, setValue, vis } = props;
+
+  return (
+    <>
+      <EuiPanel paddingSize="s">
+        <EuiTitle size="xs">
+          <h2>
+            <FormattedMessage
+              id="kbnVislibVisTypes.editors.pointSeries.settingsTitle"
+              defaultMessage="Settings"
+            />
+          </h2>
+        </EuiTitle>
+        <EuiSpacer size="s" />
+
+        <BasicOptions {...props} />
+
+        {vis.hasSchemaAgg('segment', 'date_histogram') ? (
+          <SwitchOption
+            label={i18n.translate('kbnVislibVisTypes.editors.pointSeries.currentTimeMarkerLabel', {
+              defaultMessage: 'Current time marker',
+            })}
+            paramName="addTimeMarker"
+            value={stateParams.addTimeMarker}
+            setValue={setValue}
+          />
+        ) : (
+          <SwitchOption
+            label={i18n.translate('kbnVislibVisTypes.editors.pointSeries.orderBucketsBySumLabel', {
+              defaultMessage: 'Order buckets by sum',
+            })}
+            paramName="orderBucketsBySum"
+            value={stateParams.orderBucketsBySum}
+            setValue={setValue}
+          />
+        )}
+
+        {vis.type.type === 'histogram' && (
+          <SwitchOption
+            label={i18n.translate('kbnVislibVisTypes.editors.pointSeries.showLabels', {
+              defaultMessage: 'Show values on chart',
+            })}
+            paramName="show"
+            value={stateParams.labels.show}
+            setValue={(paramName, value) =>
+              setValue('labels', { ...stateParams.labels, [paramName]: value })
+            }
+          />
+        )}
+
+        <EuiSpacer size="s" />
```

Comment:
Do we need this spacer?

Reply:
No, it looks fine without it:

<img height="200" src="https://user-images.githubusercontent.com/11524637/63024576-27a29a00-beb0-11e9-8029-b25eac3716cd.png"/>

I removed it.